flow/connectors/mysql/cdc.go

```diff
@@ -333,6 +344,7 @@ func (c *MySqlConnector) PullRecords(
 		case *replication.PreviousGTIDsEvent:
 			// TODO look into this, maybe we just do gset.Update(ev.GTIDSets)
 		case *replication.RowsEvent:
+			inTx = true
```

Comment:
Maybe this should be on QueryEvent `"BEGIN"` instead. Current logic wait for transaction when dealing with non transactional updates